font/src/ft/mod.rs

```diff
@@ -213,16 +222,25 @@ impl FreeTypeRasterizer {
         let size = Size::new(size.as_f32_pts() * self.device_pixel_ratio * 96. / 72.);
         self.pixel_size = f64::from(size.as_f32_pts());
 
+        let mut pattern = Pattern::new();
+        pattern.add_family(&desc.name);
+        pattern.add_pixelsize(self.pixel_size);
+
+        let hash = pattern.hash();
```

Comment:
Do we need to hash inside of here and then pass it? We pass the pattern anyways, right, so it would just remove one parameter to hash inside of `query_font` without any drawbacks?